guillotina/configure/__init__.py

```diff
@@ -174,7 +174,15 @@ def load_contenttype(_context, contenttype):
         behaviors=[resolve_dotted_name(b) for b in conf.get("behaviors", []) or ()],
         add_permission=conf.get("add_permission") or DEFAULT_ADD_PERMISSION,
         allowed_types=conf.get("allowed_types", None),
+        globally_addable=conf.get("globally_addable", True),
     )
+
+    # Populate app settings with globally addable types
+    if not factory.globally_addable:
```

Comment:
what is the purpose of this? I think we can remove right?

Reply:
I think this was a request by @bloodbare to be compatible with `guillotina_cms` https://github.com/plone/guillotina_cms/blob/master/guillotina_cms/content/contraintypes.py#L20

Reply:
we have this functionallity on GCMS, it can be kept there. In case we move to G should be on _settings.py and no need to setdefault.

Reply:
ahh, gotcha.

Reply:
So @lferran it sounds like this can be removed then.